### PR TITLE
Update partner programmes

### LINKED ISSUE #214: Update all partners pages to use CMS data

### PR DESCRIPTION
**Pull request by pmahnke**
## Done

- Finishes work started in #221
- Includes a fair bit of work on the partners db
- Update secondary nav spacing
- Still might require work on become-a-partner

## QA

- Check out this feature branch
- Run the site using the command `dotrun serve`
- Visit http://0.0.0.0:8002/ and click on Partners in the navigation
Navigate to http://0.0.0.0:8002/partners and see the available partner programmes. QA the links on this page
QA all the links on the following pages: http://0.0.0.0:8002/partners/public-cloud, http://0.0.0.0:8002/partners/ihv-and-oem, http://0.0.0.0:8002/partners/desktop, http://0.0.0.0:8002/partners/channel-and-reseller, http://0.0.0.0:8002/partners/gsi & http://0.0.0.0:8002/partners/become-a-partner

## Issue / Card

Fixes https://github.com/canonical-web-and-design/web-squad/issues/2559 & Fixes #214

### CODE REVIEW

**Comment by anthonydillon** [comment]
@pmahnke does this really resolve https://github.com/canonical-web-and-design/web-squad/issues/2559? If so you need to repeat the key word "Fixes" to link them. For example:
`Fixes canonical-web-and-design/web-squad#2559, Fixes #214`

File: static/sass/_pattern_lists.scss [hunk 1]
```diff
@@ -37,4 +37,13 @@
       padding: 0;
     }
   }
+
+  .p-divider.has-min-height {
```
[the remark this answers is not in the document]
Comment (pmahnke) [reply static/sass/_pattern_lists.scss:42]: Done

**Comment by nottrobin** [comment]
@pmahnke the breaking tests very helpfully caught this genuine error on `/careers/engineering`:

image

This also happens in local development. But thankfully not in live or on master, suggesting something in this branch broke that page.

**Comment by waynecrosby** [comment]
@anthonydillon Is there anything we can do about the DELL EMC logo to make it match the others (/partners/desktop & /partner/channel-and-reseller), also, AWS is massive on (/partner/channel-and-reseller)? Thanks

**Comment by anthonydillon** [comment]
@waynecrosby looks like we only have 72px version. If you can find a larger one I'll update it: https://manager.assets.ubuntu.com/?q=dell+emc

**Comment by waynecrosby** [comment]
@anthonydillon Here is a bigger version:
https://assets.ubuntu.com/v1/6ae77bbb-Dell_EMC_logo_lrg.svg
cheers

**Comment by anthonydillon** [comment]
That better? https://canonical-com-canonical-web-and-design-pr-230.run.demo.haus/partners/find-a-partner?search=dell

**Comment by waynecrosby** [comment]
Much better thanks +1

**Pull request merged**